### PR TITLE
Configurable ender quarry RF storage

### PR DESCRIPTION
**Pull request by Midnight145**
This removes the need for GTNH's modified version of EU2, where the only changes between the modified version and release are a mitigation for one of the dupes fixed in #438, and to modify the Ender Quarry's storage (set to 200,000,000 instead of 10,000,000).

Currently, it adds one new config option, and you can set the option to 0 to disable it. It defaults to the original value of 10,000,000 instead of the modified version currently in GTNH.

### CODE REVIEW

**Comment by Midnight145** [comment]
i hate spotless so much

**Comment by Midnight145** [comment]
(Briefly talked about this on Discord) 
Unsure if this is best placed in hodgepodge or somewhere else,  as this is more of a balance change than a bugfix, feel free to redirect me to somewhere else if it should go there instead.

File: src/main/java/com/mitchej123/hodgepodge/config/TweaksConfig.java [hunk 1]
```diff
@@ -199,6 +199,11 @@ public class TweaksConfig {
     @Config.DefaultInt(64)
     public static int ic2SeedMaxStackSize;
 
+    @Config.Comment("Ender Quarry RF Storage Override (0 to disable)")
```
Comment (wlhlm) [inline src/main/java/com/mitchej123/hodgepodge/config/TweaksConfig.java:202]: Maybe mention the original value? Like
```suggestion
    @Config.Comment("Ender Quarry RF Storage Override (ExU default value: 10000000) (0 to use default value)")
```

File: src/main/java/com/mitchej123/hodgepodge/mixins/late/extrautilities/MixinContainerFilingCabinet.java
Comment (wlhlm) [inline src/main/java/com/mitchej123/hodgepodge/mixins/late/extrautilities/MixinContainerFilingCabinet.java:1]: Changes look good, but please put this in a separate PR.

**Review by wlhlm — CHANGES_REQUESTED** [review]

**Review by wlhlm — APPROVED** [review]

**Pull request merged**